tidy3d/components/data/monitor_data.py

```diff
@@ -37,12 +37,13 @@
 
 from ..base_sim.data.monitor_data import AbstractMonitorData
 
-
 Coords1D = ArrayFloat1D
 
 
 class MonitorData(AbstractMonitorData, ABC):
-    """Abstract base class of objects that store data pertaining to a single :class:`.monitor`."""
+    """
+    Abstract base class of objects that store data pertaining to a single :class:`.monitor`.
```

Comment:
probably should be 
```
:class:`.Monitor`
```
(typo from before)

Reply:
Thanks! Will include this in the full review I'm going to do after going through a first pass of these comments